templates/zerver/help/add-custom-emoji.md

```diff
@@ -0,0 +1,59 @@
+# Add custom emoji
+
+If you are an administrator of a Zulip organization, you can add custom emojis
+to your organization for your organization members to use.
+
+1. Click the cog (![cog](/static/images/help/cog.png)) in the upper right corner
+of the right sidebar.
+
+2. Select **Administration** from the dropdown menu that appears.
+
+    ![Administration dropdown](/static/images/help/administration.png)
+
+3. Upon clicking **Administration**, your view will be replaced with the
+**Administration** page. You will be located on the **Organization settings**
+tab by default.
+
+    ![Administration page](/static/images/help/admin-window.png)
+
+4. Scroll down to the **Custom emoji** section located beneath the
+**Organization settings** section.
+
+    ![Custom emoji](/static/images/help/custom-emoji.png)
+
+5. In the green section labeled **Add a New Emoji**, find the **Emoji name** and
+**Emoji URL** fields.
+
+    * In the **Emoji name** field, enter a phrase for your emoji. No spaces may
+    be included in your emoji phrase.
+
+        ![Emoji name](/static/images/help/emoji-name.png)
+
+        Users will use the designated emoji phrase surrounded by colons to
+        insert your emoji into their messages.
+
+    * In the **Emoji URL** field, insert the link to your custom emoji's image. The
+    link must start with **http://** or **https://**.
+
+        ![Emoji URL](/static/images/help/emoji-url.png)
+
+        Please note that the image will be scaled down to a 25x25 icon in users' messages.
+
+6. After filling out the **Emoji name** and **Emoji URL** fields, click the blue
+**Add emoji** button to add your custom emoji to your Zulip organization.
+
+    ![Add emoji](/static/images/help/add-emoji.png)
+
+7. Upon clicking the **Add emoji** button, a section labeled **Custom emoji added!**
+will appear in the **Add a New Emoji** section, and the emoji's information and
+settings will be displayed above the **Add a New Emoji** section, confirming the
+success of your custom emoji's addition to your organization.
```

Comment:
This phrase is a tad long. Maybe try breaking it down, e.g. end the first phrase at the second comma.

What about "the addition of your custom emoji"? Genitive doesn't sound right to me here (not a native speaker though).